### PR TITLE
V2 APIs security config fix

### PR DESCRIPTION
**Pull request by raushan-appdirect**
#### Dependencies
https://github.com/AppDirect/google-connector/pull/774

#### Description
Description of the PR. Make language appropriate for other engineers not involved.

#### Manual merge checklist:
- [ ] Code Review completed
- [ ] Code coverage
- [ ] QA Validation completed
  - Testrail TestCase/Plan link:
- [ ] Approved by a PI tech lead
- [ ] Github checks all green

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <groupId>com.appdirect</groupId>
     <artifactId>service-integration-sdk</artifactId>
     <packaging>jar</packaging>
-    <version>1.124-SNAPSHOT</version>
+    <version>1.125-SNAPSHOT</version>
```
Comment (ketanmulay-appdirect) [inline pom.xml:20]: we dont need to update version manually everytime. It will be incremented automatically.

Comment (raushan-appdirect) [reply pom.xml:20]: I had run the build on this branch, the version has been incremented by CI/CD
i.e. after the merge, a release build is not required

Comment (ketanmulay-appdirect) [reply pom.xml:20]: ok

**Review by ketanmulay-appdirect — APPROVED** [review]
LGTM

**Review by nsadaphal — APPROVED** [review]
LGTM

**Pull request merged**